src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -68,16 +64,21 @@ public Long[] executeSimple(Object[][] params) {
 			.examples("floor(2.34) = 2", "floor(2) = 2", "floor(2.99) = 2")
 			.since("2.2"));
 		
-		Functions.registerFunction(new SimpleJavaFunction<Long>("round", numberParam, DefaultClasses.LONG, true) {
+		Functions.registerFunction(new SimpleJavaFunction<Number>("round", new Parameter[] {new Parameter<>("n", DefaultClasses.NUMBER, true, null), new Parameter<>("d", DefaultClasses.NUMBER, true, new SimpleLiteral<Number>(-1, false))}, DefaultClasses.NUMBER, true) {
 			@Override
-			public Long[] executeSimple(Object[][] params) {
+			public Number[] executeSimple(Object[][] params) {
 				if (params[0][0] instanceof Long)
 					return new Long[] {(Long) params[0][0]};
-				return new Long[] {Math2.round(((Number) params[0][0]).doubleValue())};
+				double value = ((Number) params[0][0]).doubleValue();
+				int placement = ((Number)params[1][0]).intValue();
+				if (placement > 0) {
```

Comment:
You can omit braces here (optional)

Reply:
Can't as the `params` array is an Object